src/vs/workbench/browser/parts/panel/panelPart.ts

```diff
@@ -281,6 +281,7 @@ export class PanelPart extends CompositePart<Panel> implements IPanelService {
 	}
 
 	private onPanelOpen(panel: IPanel): void {
+		const viewsService = this.instantiationService.invokeFunction(accessor => accessor.get(IViewsService));
```

Comment:
This can be removed I suppose as you are having `IViewService` defined in the constructor